runtime/gc_glue_java/MarkingSchemeRootClearer.cpp

```diff
@@ -306,9 +306,12 @@ MM_MarkingSchemeRootClearer::scanOwnableSynchronizerObjects(MM_EnvironmentBase *
 void
 MM_MarkingSchemeRootClearer::doMonitorReference(J9ObjectMonitor *objectMonitor, GC_HashTableIterator *monitorReferenceIterator)
 {
+	_env->getGCEnvironment()->_markJavaStats._monitorReferenceCandidates += 1;
```

Comment:
I would swap this line and line 311, var declaration/initialization should come first in a case like this

Reply:
For sure I could do the swap. I am just wondering if variable declaration/initialization always comes first.  @RSalman

Reply:
Yup, typically when the declared/init var scope is the entire function.